# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../mtcps/compile-fix.patch
# Copyright (C) 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -Nur mtcps-0.2.2-orig/mtcps.c mtcps-0.2.2/mtcps.c
--- mtcps-0.2.2-orig/mtcps.c	2005-11-19 01:31:37.000000000 +0000
+++ mtcps-0.2.2/mtcps.c	2005-12-22 14:48:47.000000000 +0000
@@ -23,6 +23,7 @@
 
 #include "sig.h"
 char *prog;
+char **environ;
 int dropname = 0;
 int loglevel = 1;
 unsigned long maxchildren = 100;
